go-controller/pkg/util/ovs.go

```diff
@@ -249,6 +251,10 @@ func SetExecWithoutOVS(exec kexec.Interface) error {
 		if err != nil {
 			return err
 		}
+		runner.ipsecPath, err = exec.LookPath(ipsecCommand)
```

Comment:
Should we only do this when ipsec is enabled ?

Reply:
I think it's ok to initialize ipsec command by default (ovnk image have ipsec binary by default).